docker_templates/templates/snippet/vcs_import.Dockerfile.em

```diff
@@ -0,0 +1,9 @@
+@[if 'vcs' in locals()]@
+@[if vcs]@
+WORKDIR @(ws)
+@[for i, (imports_name, imports) in enumerate(vcs.items())]@
```

Comment:
:+1: we'll need to keep in mind to update https://github.com/osrf/docker_images/blob/master/ros2/images.yaml.em#L48 accordingly if we don't want the logic of constructing / setting the filename in this template